docs/guides/argo-cd-crossplane.md

```diff
@@ -0,0 +1,38 @@
+---  
+title: Argo CD
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Configuring Crossplane with Argo CD
+ 
+## Overview
+ 
+[Argo CD](https://argoproj.github.io/cd/) and [Crossplane](https://crossplane.io)
+are a great combination. Argo CD provides GitOps while Crossplane turns any Kubernetes
+cluster into a Universal Control Plane for all of your resources. There are
+configuration details required in order for the two to work together properly.
+This doc will help you understand these requirements. It is recommended to us
+Argo CD version 2.4.8 or later with Crossplane.

```

Comment:
Is the version limitation driven by a specific feature in this release?

Reply:
it's the version I tested with. It probably works as of 2.2.0, but I didn't load all of those and test.

Reply:
cool, makes sense